it/v4/endpoints/bfLoss/create/CreateBFLossControllerHipISpec.scala

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v4.endpoints.bfLoss.create
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import common.errors.{RuleDuplicateSubmissionError, TypeOfLossFormatError}
+import play.api.libs.json._
+import play.api.libs.ws.{WSRequest, WSResponse}
+import play.api.test.Helpers._
+import shared.models.errors._
+import shared.models.utils.JsonErrorValidators
+import shared.services.{AuditStub, AuthStub, DownstreamStub, MtdIdLookupStub}
+import shared.support.IntegrationBaseSpec
+
+class CreateBFLossControllerHipISpec extends IntegrationBaseSpec with JsonErrorValidators {
+
+  val lossId = "AAZZ1234567890a"
+
+  val requestBody: JsValue = Json.parse(
+    """
+      |{
+      |  "businessId": "XKIS00000000988",
+      |  "typeOfLoss": "self-employment",
+      |  "taxYearBroughtForwardFrom": "2018-19",
+      |  "lossAmount": 256.78
+      |}
+    """.stripMargin
+  )
+
+  private trait Test {
+    val nino                  = "AA123456A"
+    def downstreamUrl: String = s"/itsd/income-sources/brought-forward-losses/$nino"
+
+    def setupStubs(): StubMapping
+
+    def request: WSRequest = {
+      setupStubs()
+      buildRequest(s"/$nino/brought-forward-losses")
+        .withHttpHeaders(
+          (ACCEPT, "application/vnd.hmrc.4.0+json"),
+          (AUTHORIZATION, "Bearer 123"),
+          ("suspend-temporal-validations", "true")
+        )
+    }
+
+    lazy val responseBody: JsValue = Json.parse(
+      s"""
+        |{
+        |  "lossId": "AAZZ1234567890a",
+        |  "links": [
+        |    {
+        |      "href": "/individuals/losses/$nino/brought-forward-losses/$lossId",
+        |      "method": "GET",
+        |      "rel": "self"
+        |    },
+        |    {
+        |      "href": "/individuals/losses/$nino/brought-forward-losses/$lossId",
+        |      "method": "DELETE",
+        |      "rel": "delete-brought-forward-loss"
+        |    },
+        |    {
+        |      "href": "/individuals/losses/$nino/brought-forward-losses/$lossId/change-loss-amount",
+        |      "method": "POST",
+        |      "rel": "amend-brought-forward-loss"
+        |    }
+        |  ]
+        |}
+      """.stripMargin
+    )
+
+    val downstreamResponse: JsValue = Json.parse(
+      s"""
+        |{
+        |  "lossId": "$lossId"
+        |}
+      """.stripMargin
+    )
+
+    def errorBody(code: String): String =
```

Comment:
this is the old error format. should be 
`
{
    "response": [
        "errorCode": $code",
        "errorDescription": "downstream message"
    ]
}
`